docs/quick-visual-tutorials/visual-ibac-istio-authorization-policies.mdx

```diff
@@ -8,15 +8,322 @@ import CodeBlock from "@theme/CodeBlock";
+
+</details>
+
+<details>
+<summary>Install Otterize OSS <b>with Istio watcher active</b></summary>
+
+{@include: ../_common/install-otterize-from-cloud-with-istiowatcher.md}
+
+</details>
+
+<details>
+<summary>Install the Otterize CLI</summary>
+
+{@include: ../_common/install-otterize-cli.md}
+
+</details>
+
+## Install and configure Istio
+<details>
+<summary>Install Istio in the cluster via Helm</summary>
+
+{@include: ../_common/install-istio.md}
+
+</details>
+
+<details>
+<summary>Add HTTP methods and request paths to Istio exported metrics</summary>
+
+Apply this configuration in the `istio-system` namespace, propagating it to all namespaces covered by the mesh.
+
+```
+kubectl apply -f https://docs.otterize.com/code-examples/network-mapper/istio-telemetry-enablement.yaml
+```
+
+```yaml
+{@include: ../../static/code-examples/network-mapper/istio-telemetry-enablement.yaml}
+```
+
+</details>
+
+:::tip
+HTTP request paths and methods aren't exported in Envoy's connection metrics by default, but we do want to capture those
+details when creating the network map. That way we not only have better visibility of the calling patterns,
+e.g. in the access graph, but we can also use that information to automatically generate fine-grained intents and
+enforce them with Istio authorization policies.
+:::
+
+## Deploy demo to simulate traffic
+<details>
+<summary>Create a namespace for our demo application and label it for Istio injection</summary>
+
+```bash
+kubectl create namespace otterize-visual-tutorial-istio
+kubectl label namespace otterize-visual-tutorial-istio istio-injection=enabled
+```
+</details>
+
+<details>
+<summary>Deploy the demo set of services</summary>
+
+```
+kubectl apply -n otterize-visual-tutorial-istio -f https://docs.otterize.com/code-examples/ibac-for-istio/demo-app.yaml
+```
+</details>
+
+## Seeing the access graph
+
+In the Otterize Cloud UI, your [cluster](https://app.otterize.com/clusters) should now show all 3 Otterize OSS operators &mdash; the intents operator, network mapper, and credentials operator &mdash; as connected, with a green status.
+
+And when you go back to the [access graph](https://app.otterize.com/access-graph) (and select your cluster from the dropdown, if needed), you should see the following map for the demo running in your cluster:
+
+![Access graph](/img/quick-tutorials/istio-visual-tutorial/phase-0.png)
+
+Each service is shown as a node in the access graph, while the dashed lines (edges) connecting the services show access between them, as detected by the network mapper.
+We need to adjust the view to Istio authorization policies, so enable the Istio "Use in access graph" button and disable the NetworkPolicies and Kafka toggle buttons.
+
+![Access graph](/img/quick-tutorials/istio-visual-tutorial/access-graph-panel.png)
+
+## Try out IBAC with shadow mode
+
+Now let's start to roll out access controls, but remain in shadow mode: no actual enforcement of controls, yet.
```

Comment:
Start with shadow mode value explained, we give you the power of Would be blocked